### ISSUE TITLE
Inconsistency results of MAI dataset metrics

### ISSUE THREAD

**Issue by Cynicarlos** [RenYangSCU/ISPDiffuser#10]
Hi there,
Could you explan why there is the inconsistency as follows? The second one is from FourierISP. Thank you.

<img width="1542" height="349" alt="Image" src="https://github.com/user-attachments/assets/611b780b-4af8-4c32-bb67-e8e47c1cf324" />

<img width="1668" height="329" alt="Image" src="https://github.com/user-attachments/assets/c2459214-083f-456e-8749-f3ac8a856a1f" />

**Comment by RenYangSCU** [comment]
Since the Ground Truth (GT) for the MAI test set is not publicly available, we partitioned the original training set into new training and testing subsets, following the methodology described in our paper 'ISPDiffuser: Learning RAW-to-sRGB Mappings with Texture-Aware Diffusion Models and Histogram-Guided Color Consistency'. Although we maintained the same train-test ratio as FourierISP, the specific data composition may vary. Consequently, we retrained and re-evaluated FourierISP on our split to ensure a fair comparison; this discrepancy in data partitioning likely accounts for the observed performance differences.

**Comment by Cynicarlos** [comment]
Good. But where did FourierISP mention the split of MAI dataset? I didn't see it. BTW, could you please share the ckpt of FourierISP you retrained on MAI dataset? Thanks.
